blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import {
+	Heading,
+	HeadingSection,
+	Image,
+	Link,
+	Button,
+	Paragraph,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-quilted-image";
+function QuiltedImage({ customFields }) {
+	const {
+		headline,
+		fullWidthImage,
+		image1URL,
+		image1AspectRatio,
+		overlay1Text,
+		overlay1TextVariant,
+		button1Text,
+		item1Action,
+		button1Variant,
+		image2URL,
+		image2AspectRatio,
+		overlay2Text,
+		overlay2TextVariant,
+		button2Text,
+		item2Action,
+		button2Variant,
+		image3URL,
+		image3AspectRatio,
+		overlay3Text,
+		overlay3TextVariant,
+		button3Text,
+		item3Action,
+		button3Variant,
+	} = customFields;
+
+	return (
+		<div className={BLOCK_CLASS_NAME}>
+			<HeadingSection>
+				{headline ? <Heading>{headline}</Heading> : null}
+				<div className={`${BLOCK_CLASS_NAME}--wrapper`}>
+					<Link
+						href={item1Action || "#"}
+						className={`${BLOCK_CLASS_NAME}--media-panel  ${
+							fullWidthImage === "top" ? `${BLOCK_CLASS_NAME}--wrapper--top` : ""
+						}`}
+					>
+						<Image src={image1URL} style={{ aspectRatio: image1AspectRatio }} />
+						<Stack className={`${BLOCK_CLASS_NAME}--overlay`} inline>
+							{overlay1Text ? (
+								<Paragraph className={`${BLOCK_CLASS_NAME}--overlay-text-${overlay1TextVariant}`}>
+									{overlay1Text}
+								</Paragraph>
+							) : null}
+							<Button variant={button1Variant} size="small" assistiveHidden>
+								{button1Text}
+							</Button>
```

Comment:
We should only render the button if the button text is present - the editor is showing a small button currently

```suggestion
							{button1Text ? <Button variant={button1Variant} size="small" assistiveHidden>
								{button1Text}
							</Button> : null}
```